plugins/services/src/js/structs/__tests__/PodContainer-test.js

```diff
@@ -225,66 +225,48 @@ describe("PodContainer", function() {
   describe("#hasHealthChecks", function() {
     it("returns false if no health checks defined", function() {
       const podContainer = new PodContainer({
-        endpoints: [
-          { name: "nginx", allocatedHostPort: 31001 },
-          { name: "netcat", allocatedHostPort: 31002 }
-        ]
+        conditions: []
       });
 
       expect(podContainer.hasHealthChecks()).toBeFalsy();
     });
 
     it("returns true if 'healthy' is defined for at least one container", function() {
       const podContainer = new PodContainer({
-        endpoints: [
-          { name: "nginx", allocatedHostPort: 31001, healthy: true },
-          { name: "netcat", allocatedHostPort: 31002 }
+        conditions: [
+          {
+            lastChanged: "2019-01-01T12:00:00.000Z",
+            lastUpdated: "2019-01-01T12:00:00.000Z",
+            name: "healthy",
+            reason: "health-reported-by-mesos",
+            value: "true"
+          }
         ]
       });
 
       expect(podContainer.hasHealthChecks()).toEqual(true);
     });
-
-    it("returns true if at least one check has failed", function() {
```

Comment:
why do we removing these tests?

Reply:
Because for me, they test the same scenario as the other tests - has or doesn't have health checks, so they are not needed.